site/app/templates/grading/electronic/Status.twig

```diff
@@ -590,15 +590,15 @@
         }
 
         // Get all elements with class="tablinks" and remove the class "active"
-        tablinks = document.getElementsByClassName("tablinks");
+        tablinks = document.getElementsByClassName("btn btn-primary");
         for (i = 0; i < tablinks.length; i++) {
             tablinks[i].className = tablinks[i].className.replace(" active", "");
         }
+        console.log(tablinks);
```

Comment:
Buttons work nicely now! Please remove the `console.log` and I believe it is good to merge. :)